charts/opa/values.yaml

```diff
@@ -27,7 +27,7 @@ extraEnv: []
 portOverride: ""
 
 serviceAccount:
-  create: false
+  create: true
   name: ""
```

Comment:
Is it intended to create a SA with name "default"?

Reply:
With the name of the deployment